dpnp/dpnp_iface_histograms.py

```diff
@@ -208,6 +227,189 @@ def _get_bin_edges(a, bins, range, usm_type):
     return bin_edges, None
 
 
+def _bincount_validate(x, weights, minlength):
+    if x.ndim > 1:
+        raise ValueError("object too deep for desired array")
+    if x.ndim < 1:
+        raise ValueError("object of too small depth for desired array")
+    if not dpnp.issubdtype(x.dtype, dpnp.integer) and not dpnp.issubdtype(
+        x.dtype, dpnp.bool_
```

Comment:
if you don't mind, I'd propose to use `dpnp.bool` here
```suggestion
        x.dtype, dpnp.bool
```

Reply:
Thanks! Done